en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md

```diff
@@ -0,0 +1,31 @@
+# Salesforce REST Connector Overview
+
+This documentation provides overall mapping with each sections in Salesforce connector and a inbound endpoint documentation. Click the links of the shown bellow to view the documentation for **Salesforce REST Connector** and **Salesforce Inbound Endpoint**.
+
+## Salesforce REST Connector
+
+* **[Salesforce Access Token Generation](sf-access-token-generation.md)**
+
+  This section includes how to obtain the OAuth2 tokens from Salesforce REST API.
+
+* **[Salesforce Rest API Connector Example](sf-rest-connector-example.md)**
+
+  This example explains how to use the Salesforce client to connect with the Salesforce instance and perform the **create** and **retrive** operations.
+
+* **[Salesforce Rest API Connector Configuration](sf-rest-connector-config.md)**
+
+  This documentation provides a reference guide for the Salesforce Rest API operations.
+  
+## Salesforce Inbound Endpoint  
```

Comment:
Add some information about what can be done via the endpoint.

Reply:
Added.